components/graphs/Graph.tsx

```diff
@@ -39,67 +58,101 @@ const Graph: FC<{ siteUUID: string }> = ({ siteUUID }) => {
     maxGeneration,
   ];
 
+  const renderLabel = ({ viewBox: { x }, height }: any) => {
+    const yy = height * 0.75 + 5;
+    const textProps = {
+      className: 'text-xs fill-ocf-gray-1000',
+      textAnchor: 'middle',
+      x: x + 1,
+      y: yy + 10,
+    };
+    return (
+      <g>
+        <g className="fill-white">
+          <text stroke="white" strokeWidth="1em" {...textProps}>
+            {/* Use invisible "H" character with stroke width to create white background around text */}
+            {'H'.repeat('Now'.length)}
+          </text>
+          <text {...textProps}>Now</text>
+        </g>
```

Comment:
Most changes in this PR are just to make animations work and are small (is `isLoading` before rendering graph). This is a weirder one, to add a background color of white on the text I had to create another text element and give that element a stroke width, that appears around the characters. Since the 'H' has straight lines, so does the stroke. Apparently SVG's don't do `background-color`.

Reply:
oh weird